.github/workflows/windows.yml

```
@@ -422,6 +422,9 @@ jobs:
           - name: 'API tests'
             cmd: 'tests/python_tests/test_continuous_batching.py tests/python_tests/test_generation_config.py tests/python_tests/test_sampling.py tests/python_tests/test_text_streamer.py'
             timeout: 60
+          - name: 'Rag tests'
+            cmd: 'tests/python_tests/test_rag.py'
+            timeout: 15
```

```suggestion
            timeout: 30
```
Failed in https://github.com/openvinotoolkit/openvino.genai/actions/runs/14644342760/job/41097717431?pr=2057

---

Timeout increased